### PR TITLE
Add container commands

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: mypy.ini [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [mypy]
 strict = True
+show_error_codes = True
+disable_error_code = misc
```
Comment (mdellweg) [inline mypy.ini:4]: I really don't want to do that, but it prevents us from having the two files
`file/remote.py`
`container/remote.py`

Should we rename them to `file_remote.py` and `container_remote.py`? (without the subdirectories)

Comment (daviddavis) [reply mypy.ini:4]: Adding in `__init__.py` files fixed it for me. Is that not an option?

File: pulpcore/cli/core/task.py [hunk 2]
```diff
@@ -26,21 +27,14 @@ def task(ctx: click.Context) -> None:
...
-    result = task_ctx.list(limit=limit, offset=offset, parameters=params)
-    pulp_ctx.output_result(result)
+# deepcopy to not effect other list subcommands
+list_tasks = deepcopy(list_entities)
+click.option("--name", help="List only tasks with this name.")(list_tasks)
+click.option("--name-contains", "name__contains", help="List only tasks whose name contains this.")(
+    list_tasks
+)
+click.option("--state", help="List only tasks in this state.")(list_tasks)
+task.add_command(list_tasks)
```
Comment (mdellweg) [inline pulpcore/cli/core/task.py:37]: I'm not really sure if this still justifies reusing the common `list_entries` command.

**Comment by mdellweg** [comment]
@daviddavis I decided to move the refactoring to another PR #33

**Pull request closed without merging**